docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  rookify:
+    image: "rookify-dev:latest"
+    container_name: rookify-dev
+    build:
+      context: .
+      dockerfile: DockerfileTesting
+      args:
+        - ROOKIFY_VERSION=${ROOKIFY_VERSION:-0.0.0.dev0}
+    volumes:
```

Comment:
Would it be easier to just mount the whole source directory (as described in the `README.md`) instead of pointing to individual files (like a non-existent `.id_rsa.regiocloud` private key)?

Reply:
yes thanks, I adapted that, it was confusing because the config.example where paths were a bit unpractical